components/card/collapsible/src/index.js

```diff
@@ -0,0 +1,146 @@
+import React, {Component, PropTypes} from 'react'
+import ButtonBasic from '@schibstedspain/sui-button-basic'
+import CollapsibleBasic from '@schibstedspain/sui-collapsible-basic'
+import cx from 'classnames'
+
+class CardCollapsible extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      collapsed: true
+    }
+    this._toggleCardContent = this._toggleCardContent.bind(this)
+  }
+
+  /**
+   * Changes the collapsed/expand state of the card's content.
+   */
+  _toggleCardContent () {
+    this.setState({
+      collapsed: !this.state.collapsed
+    })
+  }
+
+  /**
+   * Render a single text info item, that will be displayed in the card's header positioned depending on the expanded/collapsed state.
+   * @param label {string} Text to display.
+   * @param link {string} If set, item will be displayed as a text link.
+   * @param highlighted {boolean} Highlight the item (bold style).
```

Comment:
you could add the return tag

Reply:
Ok! I will improve the jsdoc comments in next commit.